Sources/tart/Commands/List.swift

```diff
@@ -5,18 +5,26 @@ import SwiftUI
 struct List: AsyncParsableCommand {
   static var configuration = CommandConfiguration(abstract: "List created VMs")
 
-  @Flag(name: [.short, .long], help: ArgumentHelp("Only display VM names"))
+  @Flag(name: [.short, .long], help: ArgumentHelp("Only display VM names."))
   var quiet: Bool = false
 
+  @Option(name: [.short, .long], help: ArgumentHelp("Only display VMs of this Source."))
```

Comment:
What do you think about:

* removing the short name for now, as we don't know if we may need `-s` in the future
* removing the capitalization for the word "source" and adding an example instead

```suggestion
  @Option(help: ArgumentHelp("Only display VMs from the specified source (e.g. --source local, --source oci)."))
```